components/AddForm.js

```diff
@@ -0,0 +1,95 @@
+import styled, { css } from 'styled-components';
+import Buttons from './Buttons';
+import useStore from '../hooks/useStore';
+import { useState } from 'react';
+import { nanoid } from 'nanoid';
+
+export default function AddForm() {
+  const [inputName, setInputName] = useState('');
+  const [inputInformation, setInputInformation] = useState('');
+  const addCharacter = useStore(state => state.addCharacter);
+  const toggleToast = useStore(state => state.toggleToast);
+
+  function submitForm(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    formData.append('id', nanoid());
+    const formValues = Object.fromEntries(formData);
+    addCharacter(formValues);
+    setInputName('');
+    setInputInformation('');
+    toggleToast(1, true);
+    setTimeout(() => toggleToast(1, false), 3000);
+  }
+
+  function handleReset(event) {
+    event.preventDefault();
+    setInputName('');
+    setInputInformation('');
+  }
+
+  return (
+    <StyledFormContainer
+      id="myForm"
+      onSubmit={submitForm}
+      onReset={handleReset}
+    >
+      <StyledInputField
+        required
+        type="text"
+        value={inputName}
+        name="name"
+        maxLength={20}
+        placeholder="Name"
+        onChange={event => {
+          setInputName(event.target.value);
+        }}
+      />
+      <StyledTextarea
+        required
+        type="text"
+        value={inputInformation}
+        name="information"
+        maxLength={3000}
+        rows={10}
+        placeholder="Enter here your information...
+        Attributes, Skills, Characteristics, Items, Character-Story, more..."
+        onChange={event => {
+          setInputInformation(event.target.value);
+        }}
+      />
+      <Buttons />
+    </StyledFormContainer>
+  );
+}
+
+const StyledFormContainer = styled.form`
+  display: grid;
+  gap: 20px;
+  margin: 10px;
+  padding: 10px;
+`;
+
+const StyledInputField = styled.input`
+  ${({ theme }) => css`
+    background-color: ${theme.colors.fifth};
+    font-size: ${theme.fonts.fontSizeNormal};
+    box-shadow: ${theme.boxShadow.shadow};
+  `}
+  width: 50%;
+  height: 40px;
+  padding: 5px;
+  border-radius: 10px;
+`;
+
+const StyledTextarea = styled.textarea`
+  ${({ theme }) => css`
+    background-color: ${theme.colors.fifth};
```

Comment:
This is more of a quality assurance concern, but to be very nit-picky about it: "fifth" is not really a very easily understood name for a color. I'd suggest sticking to such names as "background"/etc. The question I'd ask myself is "does someone other than myself understand what this means?" - and if I see a color name such as "fifth" in your theme, I'd have to go search for it to know where you put it, and what it does.